src/WhatsNewDialog.cpp

```diff
@@ -100,7 +105,7 @@ wxString MakeWhatsNewText()
       << wxT(R"(<p><a href=")") << WhatsNewURL << wxT(R"(">)")
       // Bug: (Windows) specified width and height should match exactly to the size of the image
       << wxT(R"(<img src="memory:whats_new_btn.jpeg" width="352" height="198" /><br></a></p>)")
-      << wxT("<h3>") << XO("What's new in Audacity %s").Format(AUDACITY_VERSION_STRING) << wxT("</h3>")
+      << wxT("<h3>") << XO("What's new in Audacity %s").Format("3.7.0") << wxT("</h3>")
```

Comment:
This looks like a leftover from testing

Reply:
Yep, kind of. I wanted a CI build for all platforms with a short version string